assets/xml/interface/week_static.xml

```diff
@@ -0,0 +1,7 @@
+<Root>
+    <File Name="week_static">
+        <Texture Name="gClockDay1stTex" OutName="clock_day_1st_tex" Format="i8" Width="48" Height="27" Offset="0x0"/>
+        <Texture Name="gClockDay2ndTex" OutName="clock_day_2nd_tex" Format="i8" Width="48" Height="27" Offset="0x510"/>
+        <Texture Name="gClockDayFinalTex" OutName="clock_day_final_tex" Format="i8" Width="48" Height="27" Offset="0xA20"/>
+    </File>
+</Root>
```

Comment:
You put this file in the interface folder, but this file is not part of the interface.csv file

Reply:
That's what's done in this PR:
https://github.com/zeldaret/mm/pull/545

Reply:
#545 does that.

Reply:
In that case I'll mark this PR as a draft until #545 is merged